src/pages/Products/Products.module.scss

```
@@ -0,0 +1,101 @@
+.Products {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 1440px;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  & h1 {
+    font-family: var(--font-family-main);
+    font-weight: 700;
+    font-size: 64px;
+    line-height: 130%;
+    margin-bottom: 20px;
+  }
+  .subtitle {
+    width: 600px;
+    height: 64px;
+    text-align: center;
+    margin-bottom: 80px;
+    font-weight: 500;
+    font: var(--font-m);
+    color: #afadb5;
+  }
+}
+.search_container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+  margin: 0 100px 50px 100px;
+}
+.search {
+  width: 1020px;
+  height: 84px;
+  padding-left: 25px;
+  padding-right: 28px;
+  box-shadow: 0px 4px 20px rgba(175, 173, 181, 0.2);
+  display: flex;
+  align-items: center;
+  gap: 18px;
+  & Input {
+    flex-grow: 1;
+  }
+}
+.products_info {
+  align-self: start;
+  margin-left: 100px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0px;
+  gap: 10px;
+  width: 356px;
+  height: 57px;
+  margin-bottom: 50px;
+
+  & h2 {
+    font-weight: 700;
+    font: var(--font-xl);
+  }
+  ._count {
+    width: 60px;
+    height: 30px;
+    color: #518581;
+    border-radius: 20px;
+    padding: 10px 16px;
+  }
+}
+.products_list {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 29px;
+  margin-left: 100px;
+  margin-right: 100px;
```

здесь и ещё где найдёшь: просто для большей красоты можно писать так:
```suggestion
  margin: 0 100px 0;
```